tests/test_DataFrameWrapper.py

```diff
@@ -141,3 +142,31 @@ def test_to_zarr_with_metadata(
     )
 
     assert hl_dataset.attrs == expected_hl_dataset_with_metadata.attrs
+
+
+@pytest.mark.parametrize(
+    argnames="out_path",
+    argvalues=(
+        str(""),
+        str("hl.zarr"),
```

Comment:
You shouldn't need to `str` a string but I think you do need to provide the `argvalues` as tuples, even if they've only got one value. e.g.:

```suggestion
        ("",),
        ("hl.zarr",),
```